Makefile

```diff
@@ -23,6 +22,8 @@ endif
 
 TARGET_BIN="$(DESTDIR)$(bindir)/$(BINARY)"
 
+KEYBINDINGS_CONF="$(DESTDIR)$(sharedir)/cosmic/com.system76.CosmicSettings.Shortcuts/v1/defaults"
```

Comment:
I wonder if we should just codify a default set of shortcuts. (Especially since some of them depend on the workspace layout choosen, no?)

Distros could still create this file to override them.

Reply:
We could do that in addition to the config, but cosmic-settings needs a way to get the on-disk config for displaying defaults.

Reply:
My gut feeling was having a `Default`-impl in `cosmic-settings-config`, is there a reason why this wouldn't work out? What happens currently if this file gets missing somehow?

Reply:
If the file is not defined, only the default bindings set in cosmic-comp will be defined right now.

Reply:
I'll look into implementing a default function

Reply:
Has this proven to be too much of a hassle?